### PR TITLE
[Dashboard] Feature: Remove analytics gating and update empty state

### PR DESCRIPTION
**Pull request by gregfromstl**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
This PR updates the dashboard redirects, layout, and icons. It also adds a new DocsButton component and updates the analytics page with new icons.

### Detailed summary
- Updated redirects for "/dashboard/connect" to "/dashboard/connect/analytics"
- Updated layout titles and paths for "Connect" section
- Added new DocsButton component
- Added new icons: DotNet, React, TypeScript, Unity, and Unreal
- Updated the analytics page with new icons and components

> The following files were skipped due to too many changes: `apps/dashboard/src/pages/dashboard/connect/analytics.tsx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/dashboard/src/core-ui/sidebar/connect.tsx [hunk 1]
```diff
@@ -13,16 +13,16 @@ type ConnectSidebarProps = {
...
-  {
-    path: "/dashboard/connect/in-app-wallets",
-    title: "In-App Wallets",
-    name: "embedded-wallets",
-  },
   {
     path: "/dashboard/connect/analytics",
     title: "Analytics",
     name: "analytics",
   },
```
Comment (joaquim-verges) [inline apps/dashboard/src/core-ui/sidebar/connect.tsx:20]: By default when I click connect tab, it still takes me to in app wallet, can you change it so it defaults to analytics?

Also I think we can get a quick win by updating the empty state here.

Maybe say something like "integrate the connect SDK in your app to view analytics." + Link to docs for each platform

Comment (gregfromstl) [reply apps/dashboard/src/core-ui/sidebar/connect.tsx:20]: Done, screenshot in the PR description

Comment (joaquim-verges) [reply apps/dashboard/src/core-ui/sidebar/connect.tsx:20]: Nice that's perfect. 

Nit: can you update the sdk  cards content to be left aligned? Cleaner imo

**Review by joaquim-verges — COMMENTED** [review]
Hmmm is there no analytics data on dev at all? Can't seem to see any even on eng@

**Review by jnsdls — APPROVED** [review]

File: apps/dashboard/src/core-ui/sidebar/connect.tsx (hunk 1 above)
Comment (jnsdls) [reply apps/dashboard/src/core-ui/sidebar/connect.tsx:20]: we'll re-visit this as part of teams migration this is functional lets ship it

**Review by jnsdls — APPROVED** [review]

**Pull request merged**